docs/rest_api.rst

```diff
@@ -378,6 +378,7 @@ Channel Management
    :statuscode 409:
     - Provided channel does not exist or
     - ``state`` and ``total_deposit`` have been attempted to update in the same request.
+   :statuscode 409 Attempt to deposit more tokens than the testing limit, or attempt to deposit lower amount than on-chain balance of the channel.
```

Comment:
one small amendement needed: Status code 409 is already defined above. Just add a new bullet point.